### DISCUSSION TITLE
对应于不同平台的修改器问题

### DISCUSSION THREAD

**Discussion by HUMHIHello** [dyang886/Game-Cheats-Manager#32] [Q&A] [answered]
Game Cheats Manager 所下载的修改器貌似不能手动选择游戏平台
以死亡细胞为例
自动下载出来的是64bit版本 也就是仅支持Epic Games Store, Xbox Game Pass的版本
不知道是否是一个小bug还是并未考虑到的一个问题?
我没有更多的测试 并不确定是不是个例现象 仅做反馈
![2024-05-29_201337](https://github.com/dyang886/Game-Cheats-Manager/assets/115995310/4616b399-542a-4aa0-9bc3-75e7311b49ed)
![2024-05-29_201539](https://github.com/dyang886/Game-Cheats-Manager/assets/115995310/8108eb10-9638-4373-895f-8a8aafd3a72c)

**Comment by dyang886** [comment, marked as answer]
经测试，官方死亡细胞修改器下载下来的压缩包内有两个版本（64 bit 和 32 bit），然而软件只会保存一个，也就是问题所在。感谢告知，下个版本会修复此问题

**Reply by HUMHIHello** [reply]
了解
辛苦你了

**Comment by tecsm** [comment]
我也遇到这个问题了，以为是修改器失效了，赞助了风灵月影还跑到官方那边申请更新修改器。后来才知道steam要用32位版本才行。跑来提交issue原来已经有了= =
